_posts/2017-19-11-connectify.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: "Connectify-WTM and GDG Baroda"
+date: 2017-11-19
+categories: ["Events"]
+tags: ["GDG Baroda", "WTM Baroda"]
+author: Tanvi Shah
+assets: "/assets/2017-11/Connectify"
```

Comment:
here you have mentioned "Connectify", note C is capital, while in the folder it is small case, i.e, "connectify". Please fix this issue.